packages/react/src/blocks/FileBlockContent/fileBlockHelpers.tsx

```diff
@@ -0,0 +1,259 @@
+import { FileBlockConfig } from "@blocknote/core";
+import { ReactNode, useCallback, useEffect, useState } from "react";
+import { RiFile2Line } from "react-icons/ri";
+import { ReactCustomBlockRenderProps } from "../../schema/ReactBlockSpec";
+
+export const DefaultFilePreview = (
+  props: Omit<
+    ReactCustomBlockRenderProps<FileBlockConfig, any, any>,
+    "contentRef"
+  >
+) => (
+  <div
+    className={"bn-file-default-preview"}
+    contentEditable={false}
+    draggable={false}>
+    <RiFile2Line size={24} />
+    {props.block.props.name}
+  </div>
+);
+
+export const FileAndCaptionWrapper = (
+  props: Omit<
+    ReactCustomBlockRenderProps<FileBlockConfig, any, any>,
+    "contentRef"
+  > & {
+    children: ReactNode;
+  }
+) => {
+  return (
+    <div className={"bn-file-and-caption-wrapper"}>
+      {props.children}
+      {props.block.props.caption && (
+        <p
+          className={"bn-file-caption"}
+          style={{
+            paddingBlock: props.block.props.caption ? "4px" : undefined,
```

Comment:
- props.block.props.caption is always defined here.
- better to move to stylesheet anyway?